todo_app/data/database_client.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pymongo
+
+from todo_app.data.item import Item
+
+def get_db():
+    client = pymongo.MongoClient(f'{os.getenv("DATABASE_CONNECTION_STRING")}')
+    return client[f'{os.getenv("DATABASE_NAME")}']
+
+def get_all_items():
+    db = get_db()
+    collection =  db.items
```

Comment:
Given that we always refer to the same collection, maybe we could just commonise this into `collection = get_collection()`